support/cas-server-support-pm-webflow/src/main/java/org/apereo/cas/pm/web/flow/PasswordManagementWebflowConfigurer.java

```diff
@@ -36,6 +40,12 @@
     private static final String PASSWORD_CHANGE_ACTION = "passwordChangeAction";
     private static final String SEND_PASSWORD_RESET_INSTRUCTIONS_ACTION = "sendInstructions";
 
+    private static final String CHECK_FOR_PASSWORD_RESET_TOKEN_ACTION = "checkForPswdResetToken";
```

Comment:
Does it make sense for these consts to go into `CasWebflowConstants`?

Reply:
Yeah, I wasn't sure on that. I've noticed a few cases of module specific actions being in the modules, so I erred on the side of fewer files touched. But for extensibility I'd prefer them in CasWebflowConstants. If you have no objections I'll move them over...